src/crypto/node/crypto.ts

```diff
@@ -14,32 +14,100 @@
 
 import * as crypto from 'crypto';
 import {Crypto} from '../crypto';
+import * as base64js from 'base64-js';
```

Comment:
So this is cool and all, but the real test would be to delete this file, and rely on the browser context one we've already written :) That is already using SubtleCrypto, so if node is API compatible it should slide right in.  This PR could just be a deleted file!